src/azul/logging.py

```diff
@@ -20,16 +56,27 @@ def configure_test_logging(*loggers):
 
 
 def _configure_non_app_logging(*loggers):
-    logging.basicConfig(format="%(asctime)s %(levelname)-7s %(threadName)s: %(message)s")
+    non_app_handler = logging.StreamHandler()
+    non_app_handler.addFilter(LambdaLoggerFilter())
+    logging.basicConfig(
+        format=log_format,
+        datefmt=log_date_format,
+        handlers=[non_app_handler])
     _configure_log_levels(*loggers)
 
 
+def _configure_es_logging():
+    es_trace_logger = logging.getLogger('elasticsearch')
+    es_trace_logger.setLevel(logging.DEBUG)
+
+
 def _configure_log_levels(*loggers):
     azul_level_ = azul_log_level()
     root_level = root_log_level()
     logging.getLogger().setLevel(root_level)
     for logger in {*loggers, azul.log}:
         logger.setLevel(azul_level_)
+    _configure_es_logging()
```

Comment:
Placing this method here makes it a side effect of setting the level, consider adding it to the `_configure_non_app_logging/_configure_app_logging` methods.

Also the log level for ES should be effected by `_configure_log_levels`

Reply:
Can I just include the code in `_configure_es_logging` in `_configure_log_levels` since all that is occurring is a singular log level being set? That way I can set the log levels for ES using the already existing for loop.